src/Symfony/Component/Cache/Adapter/ApcuAdapter.php

```diff
@@ -37,9 +37,9 @@ public function __construct($namespace = '', $defaultLifetime = 0, $version = nu
         if (null !== $version) {
             CacheItem::validateKey($version);
 
-            if (!apcu_exists($version.':'.$namespace)) {
+            if (!apcu_exists($version.'@'.$namespace)) {
```

Comment:
Just because `:` is a nice separator for namespace, thus had to change here. `@` looks good also for version :)